hazelcast/src/main/java/com/hazelcast/internal/networking/nio/iobalancer/IOBalancer.java

```diff
@@ -71,16 +74,16 @@
     private final LoadTracker inLoadTracker;
     private final LoadTracker outLoadTracker;
     private final String hzName;
-    private final BlockingQueue<Runnable> workQueue = new LinkedBlockingQueue<Runnable>();
+    private final BlockingQueue<Runnable> workQueue = new LinkedBlockingQueue<>();
     private volatile boolean enabled;
     private IOBalancerThread ioBalancerThread;
 
     // only IOBalancerThread will write to this field.
-    @Probe
+    @Probe(name = NETWORKING_METRIC_NIO_IO_BALANCER_IMBALANCE_DETECTED_COUNT, unit = COUNT)
```

Comment:
Don't we use `COUNT` by the default? Same question for another probe in this class.

Reply:
Removed the unit.